player/drm/css/style.css

```diff
@@ -119,6 +119,10 @@ body .demo-detail .code-example {
     margin-top: 47px;
 }
 
+#bitmovinplayer-video-player-container, #player-container {
```

Comment:
Why don't we select all children of the `player-container`? Doing that seems to work as well, and it also retains the rounded edges.

Reply:
https://github.com/bitmovin/demos/pull/49/commits/2ca1f3fb9403800aa87669eae17dd036f7ba11d1 Agree, tested it out and indeed we keep the edges and playback works! Thanks for the suggestion!